WalletWasabi/JsonConverters/KeyPathJsonConverter.cs

```diff
@@ -25,11 +19,9 @@ public override bool CanConvert(Type objectType)
 	}
 
 	/// <inheritdoc />
-	public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+	public override void WriteJson(JsonWriter writer, KeyPath? value, JsonSerializer serializer)
 	{
-		var kp = (KeyPath)value;
-
-		var s = kp.ToString();
+		var s = value?.ToString() ?? throw new ArgumentNullException(nameof(value));
```

Comment:
@yahiheb On L14 seems to prove that one can read `null` but we cannot write it? I don't know if it is correct or not but you should explain it imo.

Reply:
I think it does not make sense to write null here, but honestly Idk.